app/lib/time_limit_config.rb

```diff
@@ -37,6 +37,10 @@ def self.replace_referee_by
     14
   end
 
+  def self.additional_reference_chase_calendar_days
+    28
```

Comment:
The requirement says 20 business days. Are we doing it this way to avoid all the Covid-19 business time 'holidays'?

Reply:
Yes, seems good to me